terraform/main.tf

```diff
@@ -35,6 +35,16 @@ provider "confluent" {
 
 provider "aws" {
   region = var.region
+
+  default_tags {
+    tags = {
+      cflt_managed_by  = "terraform"
+      cflt_managed_id  = "demo-stream-designer"
+      cflt_service     = "demo-stream-designer"
+      cflt_partition   = "commercial"
+      cflt_environment = "prod"
+    }
+  }
```

Comment:
Instead lets create a new optional var of type map(string), and use that var here (call it `tags` not `cflt_tags`) - this at least exposes the ability to set tags via a variable without exposing out internal tagging conventions.